IdentityCore/tests/MSIDLastRequestTelemetryTests.m

```diff
@@ -217,6 +217,8 @@ - (void)testSaveToDisk_whenMultipleSavesThenReset_shouldOverwriteAndRestoreToSam
     
     XCTAssertEqualObjects([restoredTelemetryObject telemetryString], [telemetryObject telemetryString]);
     
+    //[[MSIDLastRequestTelemetry sharedInstance] setValue:@0 forKey:@"silentSuccessfulCount"];
+    //[[MSIDLastRequestTelemetry sharedInstance] setValue:nil forKey:@"errorsInfo"];
```

Comment:
Please confider to remove commented out code

Reply:
+1